Entity/ProductAttribute.php

```diff
@@ -38,31 +33,11 @@ class ProductAttribute
     private $attribute;
 
     /**
-     * Set value
-     *
-     * @param string $value
-     * @return ProductAttribute
+     * @var \Sulu\Bundle\ProductBundle\Entity\AttributeValue
```

Comment:
replace `\Sulu\Bundle\ProductBundle\Entity\AttributeValue` with `AttributeValue` in whole file